types/README.md

```diff
@@ -48,6 +48,10 @@ This document describes the format of the container definition files contained i
   `static types::st::Unit<DB> getSizeType(const T& container, ST returnArg)`
   where `ST` defines the combined static type of each supplied processor.
 
+- `extra`
+
+  Any extra C++ code to be included directly.
```

Comment:
nit: can this mention that there's no namespacing on this code? it might catch people unaware that they can't collide with any other container implementation.